.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+version: 2
+updates:
+  - package-ecosystem: npm
+    directory: "/"
+    schedule:
+      interval: monthly
+    open-pull-requests-limit: 10
```

Comment:
will it work for all package.json in modules directories or do we have to define each one separately ?

Reply:
created issue https://linear.app/skynetlabs/issue/SKY-782/determine-how-dependabot-works-for-subpackage-packagejson-files